README-install.md

```diff
@@ -67,13 +67,13 @@ For RHEL, you must enable the Extras and the rhel-7-fast-datapath-rpms channels
 ViaQ on Origin requires these [Yum Repos](centos7-viaq.repo).
 You will need to install the following packages: docker, iptables-services.
 
-    # yum install docker iptables-services
+        # yum install docker iptables-services
 
 You will need to configure sudo to not require a tty.  For example, create a
 file like `/etc/sudoers.d/999-cloud-init-requiretty` with the following contents:
 
-    # cat /etc/sudoers.d/999-cloud-init-requiretty
-    Defaults !requiretty
+        # cat /etc/sudoers.d/999-cloud-init-requiretty
+        Defaults !requiretty
```

Comment:
these look ok now, and do not appear to need extra indentation: https://github.com/ViaQ/Main/blob/master/README-install.md#provisioning-a-machine-to-run-viaq